sql/catalyst/src/main/scala/org/apache/spark/sql/types/AbstractDataType.scala

```diff
@@ -151,7 +151,7 @@ abstract class NumericType extends AtomicType {
 }
 
 
-private[sql] object NumericType extends AbstractDataType {
+private[spark] object NumericType extends AbstractDataType {
```

Comment:
Because it's now used in [Predictor.scala]{https://github.com/apache/spark/pull/10355/files#diff-5fe763c1d5d2d36c0154204028cd3585R56}